### PR TITLE
changes for 2.8

### PR DESCRIPTION
**Pull request by jc-berger**
For 2.8 

See Jira: https://issues.redhat.com/browse/ACM-8258

See PR for 2.9: https://github.com/stolostron/rhacm-docs/pull/5705

### CODE REVIEW

File: governance/policy_ctrl_adv_config.adoc [hunk 1]
```diff
@@ -59,7 +59,13 @@ spec:
 [#configure-debug-log] 
 == Configure debug log 
 
-Configure debug logs for each policy controller. To receive the second level of debugging information for the Kubernetes configuration controller, add the `log-level` annotation with the value of `2` to the `ManagedClusterAddOn` custom resource. By default, the `log-level` is set to `0`, which means you receive informative messages. View the following example:  
+When you configure and collect debug logs for each policy controller, the default log level might take up more space than you want. In {product-title-short} 2.9, you can reduce the volume of debug logs. You can reduce the volume below the default setting.
```
Comment (JustinKuli) [inline governance/policy_ctrl_adv_config.adoc]: This is the 2.8 doc, but mentions 2.9 here. Just something to remember (I think there are comments on the other PR that will have changes you may bring here anyway)

File: release_notes/whats_new.adoc [hunk 2]
```diff
@@ -73,6 +73,8 @@ For other Application topics, see link:../applications/app_management_overview.a
 [#governance-whats-new]
 == Governance
 
+* You can now increase or reduce the volume of debug logs from the default settings. For more details, see link:../governance/policy_ctrl_adv_config.adoc#configure-debug-log[Configure debug log].
```
Comment (dockerymick) [inline release_notes/whats_new.adoc]: What's new entry isn't required for version 2.8 bc The feature, which fixed a bug, was added in 2.9. Only governance/policy_ctrl_adv_config.adoc needs to be updated

**Review by dockerymick — APPROVED** [review]
Remove the what's new entry only for version 2.8, and apply similar changes from your previous PR. Otherwise this looks good

File: governance/policy_ctrl_adv_config.adoc [hunk 3]
```diff
@@ -59,7 +59,18 @@ spec:
 [#configure-debug-log] 
 == Configure debug log 
 
-Configure debug logs for each policy controller. To receive the second level of debugging information for the Kubernetes configuration controller, add the `log-level` annotation with the value of `2` to the `ManagedClusterAddOn` custom resource. By default, the `log-level` is set to `0`, which means you receive informative messages. View the following example:  
+When you configure and collect debug logs for each policy controller, you can adjust the log level
```
Comment (JustinKuli) [inline governance/policy_ctrl_adv_config.adoc]:
```suggestion
When you configure and collect debug logs for each policy controller, you can adjust the log level.
```

**Review by JustinKuli — COMMENTED** [review]
Other than one tiny thing, looks good!

**Review by JustinKuli — APPROVED** [review]
LGTM

**Pull request merged**